tests/test_highlevel.py

```diff
@@ -1523,7 +1523,6 @@ def convert(v):
             self.assertEqual(str(node.is_sample()), splits[1])
             self.assertEqual(convert(node.time), splits[2])
             self.assertEqual(str(node.population), splits[3])
-            self.assertEqual(msprime.text_encode_metadata(node.metadata), splits[4])
```

Comment:
Should replace this test rather than just delete it I think. We could defined a local function (in the test module) called ``base64_encode`` which will have the same definition as the old ``text_encode_metadata`` function.